### PR TITLE
Update cargo-raze to latest and regenerate artifacts.

### PR DESCRIPTION
**Pull request by PiotrSikora**
Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

**Comment by PiotrSikora** [comment]
Blocked on #46.

**Review by gbrail — APPROVED** [review]
I tested locally with this PR and #46 merged together, and indeed it works, so I'll pre-approve this even though in its current form it doesn't build ;-)

**Comment by PiotrSikora** [comment]
> I tested locally with this PR and #46 merged together, and indeed it works, so I'll pre-approve this even though in its current form it doesn't build ;-)

Thanks! FYI, I cannot merge without approval and passed tests (well, I could if I changed repo settings, but not by accident).

**Pull request merged**
